scripts/build

```diff
@@ -98,21 +102,41 @@ if [ ${ARCH} = s390x ]; then
     export GOARCH="s390x"
 fi
 
-rm -f \
-    bin/k3s-agent \
-    bin/k3s-server \
-    bin/k3s-token \
-    bin/k3s-etcd-snapshot \
-    bin/k3s-secrets-encrypt \
-    bin/k3s-certificate \
-    bin/k3s-completion \
-    bin/kubectl \
-    bin/crictl \
-    bin/ctr \
-    bin/containerd \
-    bin/containerd-shim \
-    bin/containerd-shim-runc-v2 \
-    bin/runc
+k3s_binaries=(
+    "bin/k3s-agent"
+    "bin/k3s-server"
+    "bin/k3s-token"
+    "bin/k3s-etcd-snapshot"
+    "bin/k3s-secrets-encrypt"
+    "bin/k3s-certificate"
+    "bin/k3s-completion"
+    "bin/kubectl"
+    "bin/crictl"
+    "bin/ctr"
+)
+
+containerd_binaries=(
+    "bin/containerd"
```

Comment:
With the [commit](https://github.com/k3s-io/k3s/commit/607cbf0ad6f6675c94ed436723892997f9e819d5) in May, containerd was moved from a standalone executable to a built-in part of K3S. However, in this build script for Windows, neither the standalone version (with TAG ctrd) nor the built-in version of containerd is built.

Reply:
So far the built-in containerd does not work on Windows. I've sent a [PR](https://github.com/k3s-io/k3s/pull/7973) to fix it

Reply:
Could we perhaps just roll that PR into this one?

Reply:
@seanyen, could you please cherry-pick the commit in my [PR](https://github.com/k3s-io/k3s/pull/7973), and add ctrd into the TAGS for building K3S, to enable built-in containerd?

Reply:
The PR is integrated now. Thanks for the pointers!